python/ray/includes/global_state_accessor.pxi

```diff
@@ -57,3 +58,15 @@ cdef class GlobalStateAccessor:
 
     def get_node_resource_info(self, node_id):
         return self.inner.get().GetNodeResourceInfo(CClientID.FromBinary(node_id.binary()))
+
+    def get_worker_table(self):
+        return self.inner.get().GetAllWorkerInfo()
```

Comment:
We should release the GIL when calling into C++. I forgot this in the previous PRs. Can you change them?